### PR TITLE
update mathcomp packages for 1.10.0

### PR DESCRIPTION
**Pull request by ybertot**

### CODE REVIEW

File: released/packages/coq-mathcomp-algebra/coq-mathcomp-algebra.1.10.0/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+name: "coq-mathcomp-algebra"
+version: "1.10.0"
```
Comment (clarus) [inline released/packages/coq-mathcomp-algebra/coq-mathcomp-algebra.1.10.0/opam]: I would remove the `name` and `version` from the package content as this is redundant with the file name.

File: released/packages/coq-mathcomp-algebra/coq-mathcomp-algebra.1.10.0/opam [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+maintainer: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
+
+homepage: "https://math-comp.github.io/"
+bug-reports: "https://github.com/math-comp/math-comp/issues"
+dev-repo: "git+https://github.com/math-comp/math-comp.git"
+license: "CeCILL-B"
+
+build: [ make "-C" "mathcomp/algebra" "-j" "%{jobs}%" ]
+install: [ make "-C" "mathcomp/algebra" "install" ]
+depends: [ "coq-mathcomp-fingroup" { = "1.10.0" } ]
```
Comment (clarus) [inline released/packages/coq-mathcomp-algebra/coq-mathcomp-algebra.1.10.0/opam]: I think there can be also:
```suggestion
depends: [ "coq-mathcomp-fingroup" { = version } ]
```

**Comment by ybertot** [comment]
@clarus I took your suggestions into account.  Tell me if this is right.

**Comment by clarus** [comment]
Yes! Waiting for the CI.

**Comment by clarus** [comment]
Done thanks

**Pull request merged**